src/planner/plannodes/csv_scan_plan_node.cpp

```diff
@@ -5,7 +5,7 @@
 namespace terrier::planner {
 
 common::hash_t CSVScanPlanNode::Hash() const {
-  common::hash_t hash = AbstractPlanNode::Hash();
+  common::hash_t hash = AbstractPlanNode::Hash();  // NOLINT: bugprone-parent-virtual-call: this appears intentional?
```

Comment:
Could @GustavoAngulo or @wenxuanqiu weigh in?

Reply:
I think it actually is a legit bug, now that I look at the explanation a bit more.

https://clang.llvm.org/extra/clang-tidy/checks/bugprone-parent-virtual-call.html

Will fix.